tests/testthat/test-content.R

```diff
@@ -180,3 +180,48 @@ without_internet({
     )
   })
 })
+
+with_mock_api({
+  test_that("content_render() calls the correct endpoint, returns task on success", {
+    client <- Connect$new(server = "https://connect.example", api_key = "not-a-key")
+    x <- content_item(client, "951bf3ad-82d0-4bca-bba8-9b27e35c49fa")
```

Comment:
This is minor, but this GUID is the full one, but then we return the shortend version in the mocks. Should we also shorten these here too so they are consistent?

Reply:
Yeah, the code in `setup.R` automatically shortens it. I'll shorten it in tests too.

Reply:
Most of the other tests use full-length GUIDs. Part of me feels like it's reasonable to leave them full-length because it's easier to visually parse them as a GUID. I'm not sure the reasoning behind truncating them in tests.

Reply:
Hmm, I've been finding that leaving as full-length GUIDs means that I can easily recreate a live content item in my local environment, which means that recreating test scenarios or capturing additional responses is much easier.

Reply:
I'm fine leaving them long, but maybe we should _not_ shorten them in `setup.R` then. There is definitely a tension here between making these tests easy/quick to get from a running setup vs. looking nice / easy(er) to digest when coding. But we should be consistent with which we do so that we don't run into a situation where doing something like the following would fail paradoxically:

```
GUID <- "951bf3ad-82d0-4bca-bba8-9b27e35c49fa"
x <- content_item(client, GUID)
expect_true(x$guid, GUID)
```

Reply:
The shortening in setup.R is about just to avoid running into the "long file paths" R CMD check error.

Reply:
Unsure how to resolve this; I'm going to merge and leave this as tension to resolve in the future. It's somewhat confusing, but not unmanageably so. Leaving this conversation unresolved.